smokeTest.js

```diff
@@ -0,0 +1,129 @@
+const { helpers } = require('brave-alert-lib')
+const axios = require('axios').default
+const XETHRU_STATE = require('./SessionStateXethruEnum.js')
+
+const MOV_THRESHOLD = 17
+const IM21_DOOR_STATUS = require('./IM21DoorStatusEnum')
+
+const testLocation1Id = 'SmokeTestLocation'
+const door_coreID = 'door_coreID'
+const radar_coreID = 'radar_coreID'
+
+function getRandomInt(minValue, maxValue) {
+  const min = Math.ceil(minValue)
+  const max = Math.floor(maxValue)
+  return Math.floor(Math.random() * (max - min) + min) // The maximum is exclusive and the minimum is inclusive
+}
+
+function getRandomArbitrary(min, max) {
+  return Math.random() * (max - min) + min
+}
+
+const destinationURL = process.argv[1]
+const recipientNumber = process.argv[2]
+const twilioNumber = process.argv[3]
+
+function sleep(millis) {
+  return new Promise(resolve => {
+    setTimeout(resolve, millis)
+  })
+}
+axios.defaults.headers.post['Content-Type'] = 'application/x-www-form-urlencoded'
```

Comment:
I feel like there should perhaps be another line break above these axios guys but that won't affect it working or not 😅